src/gallery/slider.js

```diff
@@ -53,10 +53,14 @@ const clientWidth = window.innerWidth,
 
 	renderImageInfo = ( mediaInfo, image, lang ) => {
 		const getImageDescription = () => {
-				if ( image.caption ) {
-					return image.caption
-				} else if ( mediaInfo.description ) {
+				// description list order
```

Comment:
should we delete the comments?

Reply:
i think it's better to keep as the memory